### PR TITLE
Added missing vb sample.

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: docs/standard/using-linq.md [hunk 1]
```diff
@@ -328,6 +328,22 @@ public static bool PublicInstancePropertiesEqual<T>(this T self, T to, params st
 }
 ```
 
+```vb
+<System.Runtime.CompilerServices.Extension()> 
+Public Shared Function PublicInstancePropertiesEqual(Of T As Class)(self As T, [to] As T, ParamArray ignore As String()) As Boolean
```
Comment (rpetrusha) [inline docs/standard/using-linq.md]:
```suggestion
Public Function PublicInstancePropertiesEqual(Of T As Class)(self As T, [to] As T, ParamArray ignore As String()) As Boolean
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thank you for adding a Visual Basic version of this example, @Youssef1313. Extensions methods, though, must be in modules, where the `Shared` keyword is not supported.

**Comment by Youssef1313** [comment]
> 
> 
> Thank you for adding a Visual Basic version of this example, @Youssef1313. Extensions methods, though, must be in modules, where the `Shared` keyword is not supported.

Thanks, I've committed your change :)

**Comment by rpetrusha** [comment]
Thanks for the additional change, @Youssef1313. I'll merge your PR now.

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**